blog/2024-11-01-reauth-reconfigure-entry-id.md

```diff
@@ -0,0 +1,13 @@
+---
+author: epenet
+authorURL: https://github.com/epenet
+title: "Reauth and reconfigure flows need to be linked to a config entry"
+---
+
+Starting a reauth or a reconfigure flow without a link to the config entry has been deprecated.
```

Comment:
Add an example of how a flow is started correctly after this statement.

Reply:
Then say what custom integrations need to do and how long time the deprecation period is.